### PR TITLE
Adds AudienceNetworkAds.initialize call to each of the ad formats

### PR DESCRIPTION
**Pull request by xilosada**

### CODE REVIEW

File: FacebookAudienceNetwork/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ## Changelog
+
+  * 5.0.1.0-beta
+    * Added calls to Facebook SDK initialization method for each ad format.
```
Comment (chauduyphanvu1) [inline FacebookAudienceNetwork/CHANGELOG.md]: I'd rephrase it to `Initialize the Facebook Audience Network SDK [...]`

File: FacebookAudienceNetwork/com/mopub/mobileads/FacebookRewardedVideo.java [hunk 2]
```diff
@@ -63,6 +66,9 @@ protected boolean checkAndInitializeSdk(@NonNull Activity launcherActivity, @Non
 
     @Override
     protected void loadWithSdkInitialized(@NonNull Activity activity, @NonNull Map<String, Object> localExtras, @NonNull Map<String, String> serverExtras) throws Exception {
+        if(!sIsInitialized.getAndSet(true)) {
+            AudienceNetworkAds.initialize(activity);
```
Comment (chauduyphanvu1) [inline FacebookAudienceNetwork/com/mopub/mobileads/FacebookRewardedVideo.java]: `loadWithSdkInitialized ` assumes that mediated SDKs have been initialized. Is this call async? If so, I'd call it in MoPub's `checkAndInitializeSdk()`.

Comment (xilosada) [reply FacebookAudienceNetwork/com/mopub/mobileads/FacebookRewardedVideo.java]: @chauduyphanvu1 The call is not async, but it seems that `checkAndInitializeSdk()` is the right place to do it. I moved the call to this method. Is it ok?

File: FacebookAudienceNetwork/CHANGELOG.md (hunk 1 above)
Comment (xilosada) [reply FacebookAudienceNetwork/CHANGELOG.md]: Updated
